converter/k8s.go

```diff
@@ -27,6 +27,7 @@ func NewK8sManifestsFromPatternfile(patternFile *pattern.PatternFile) (string, e
 	buf := bytes.NewBufferString("")
 
 	enc := yaml.NewEncoder(buf)
+	enc.SetIndent(4)
```

Comment:
Can we please use an indentation of two spaces? This will keep lines from wrapping sooner that they otherwise would, making the yaml more readable in my opinion.
```suggestion
	enc.SetIndent(2)
```

Reply:
Agreed!

Reply:
Noted! That makes sense. Thank you for pointing it out.